modes/ivy/evil-collection-ivy.el

```diff
@@ -37,8 +37,10 @@
 ;;;###autoload
 (defun evil-collection-ivy-setup ()
   "Set up `evil' bindings for `ivy-mode'."
-  (evil-collection-define-key nil 'ivy-mode-map
-    (kbd "<escape>") 'minibuffer-keyboard-quit)
+
+  (evil-collection-move-bindings-to-emacs-state 'ivy-mode-map
```

Comment:
When this is done, it looks like the unbinding gets us to ivy's default keyboard quit which seems to be 3 escape keys instead of the 1 we're expecting.